src/main/java/com/hubspot/jackson/datatype/protobuf/ProtobufDeserializer.java

```diff
@@ -337,6 +339,42 @@ protected List<Object> readArray(
     }
   }
 
+  private int parseInt(FieldDescriptor field, JsonParser parser, DeserializationContext context) throws IOException {
+    if (isUnsigned(field.getType())) {
+      long longValue = _parseLongPrimitive(parser, context);
+      if (longValue < Integer.MIN_VALUE || longValue > 0xFFFFFFFFL) {
```

Comment:
We use `0xFFFFFFFFL` in a few places. Is it worth putting it in a constant?

Reply:
👍  fd6dd816064f8b973876e744e98678877cdf956b